cabot/cabotapp/utils.py

```diff
@@ -1,6 +1,12 @@
 from django.conf import settings
+from datetime import datetime
 
 
 def build_absolute_url(relative_url):
     """Prepend https?://host to a url, useful for links going into emails"""
     return '{}://{}{}'.format(settings.WWW_SCHEME, settings.WWW_HTTP_HOST, relative_url)
+
+
+def format_datetime(dt):
+    '''Convert datetime to string. None is converted to empty string.'''
+    return '' if dt is None else datetime.strftime(dt, '%Y-%m-%d %H:%M:%S')
```

Comment:
Maybe a comment on the intended use for `format_datetime` vs `format_timestamp` from extra.py, since both convert a datetime to a string?

Reply:
Ahh, didn't realize we had two nearly-identical date formatting functions.  We should likely clean that up (i.e. rename them to something more similar).  But anyway for now I updated the comment as you suggest.